src/chainstate/burn/db/sortdb.rs

```diff
@@ -2667,6 +2733,11 @@ impl SortitionDB {
                         let tx = self.tx_begin()?;
                         SortitionDB::apply_schema_3(&tx.deref())?;
                         tx.commit()?;
+                    } else if version == "3" {
+                        // add the tables of schema 4, but do not populate them.
+                        let tx = self.tx_begin()?;
+                        SortitionDB::apply_schema_4(&tx.deref())?;
+                        tx.commit()?;
```

Comment:
Don't forget to also update ` is_db_version_supported_in_epoch()` as well!

Reply:
Just did - does it make sense to set this to "4" for epoch 2.1?